debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java

```diff
@@ -93,7 +93,7 @@ protected String zookeeperConnection() {
      */
     protected void populateDefaultConfiguration(Properties props) {
         config.setProperty(KafkaConfig.NumPartitionsProp(), String.valueOf(1));
-        config.setProperty(KafkaConfig.LogFlushIntervalMessagesProp(), String.valueOf(1));
+        config.setProperty(KafkaConfig.LogFlushIntervalMessagesProp(), String.valueOf(10_000));
```

Comment:
Why 10000 was chosen as value ? Is there any rationale behind it ?

Reply:
Damn, the 10000 was default value in the past

Reply:
Today it's 9223372036854775807 :-)